app/views/global_user_roles/edit.html.haml

```diff
@@ -0,0 +1,27 @@
+= content_for :h1 do
+  = t("pages.global_settings")
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_global", sidenav_tab: "global_user_roles"
+
+%h2= t("pages.global_user_roles")
+
+= form_for(:user_role, url: global_user_role_path(@user.id), method: :put) do |form|
+  = form.error_messages
+  %ul.form
+    %li
```

Comment:
This is nitpicky, and I realize we have it as well on the facility staff UI, but can we get rid of the `li` or at least the bullets?